docs/running-on-kubernetes.md

```diff
@@ -117,6 +117,37 @@ If the local proxy is running at localhost:8001, `--master k8s://http://127.0.0.
 spark-submit. Finally, notice that in the above example we specify a jar with a specific URI with a scheme of `local://`.
 This URI is the location of the example jar that is already in the Docker image.
 
+## Client Mode
+
+Starting with Spark 2.4.0, it is possible to run Spark applications on Kubernetes in client mode. When running a Spark
+application in client mode, a separate pod is not deployed to run the driver. When running an application in
```

Comment:
could we add a bit here after `a separate pod is not deployed to run the driver` to say that the client/driver could be outside k8s or in k8s/in a pod?